shipping/src/main/java/works/weave/socks/ShippingController.java

```diff
@@ -27,7 +27,11 @@ public String getShippingById(@PathVariable String id) {
     @ResponseBody
     Shipment postShipping(@RequestBody Shipment shipment) {
         System.out.println("Adding shipment to queue...");
-        rabbitTemplate.convertAndSend("shipping-task", shipment);
+        try {
+            rabbitTemplate.convertAndSend("shipping-task", shipment);
+        } catch (Exception e) {
```

Comment:
Might as well catch the specific Exception here (AmqpException). I guess this will get re-worked as per https://github.com/weaveworks/weaveDemo/issues/102